requirements/requirements.in

```diff
@@ -23,6 +23,7 @@ elasticsearch==1.9.0
 eulxml==1.1.3
 ghdiff==0.4
 gunicorn
+leven==1.0.4
```

Comment:
The install instructions say it needs Cython and C++ compiler. I assume we have these on our prod machines but might be worth checking.